improver/api/__init__.py

```diff
@@ -86,7 +86,7 @@
     "OccurrenceBetweenThresholds": "improver.between_thresholds",
     "OccurrenceWithinVicinity": "improver.utilities.spatial",
     "OpticalFlow": "improver.nowcasting.optical_flow",
-    "OrographicEnhancement": "improver.orographic_enhancement",
+    "MetaOrographicEnhancement": "improver.orographic_enhancement",
```

Comment:
Logic from improver/cli/orographic_enhancement.py incorporated into a new `MetaOrographicEnhancement`, which in turn, calls `OrographicEnhancement`.